src/Playground.res

```diff
@@ -1666,16 +1716,14 @@ let make = (~versions: array<string>) => {
     | Problems => "Problems"
     | Settings => "Settings"
     }
-    let onMouseDown = evt => {
+    let onClick = evt => {
```

Comment:
Using `onClick` instead of `onMouseDown` makes the tabs keyboard accessible - buttons with a click handler can be triggered with `Enter`/`Space`.

Reply:
This might have been an optimisation so that `event.preventDefault()` gets invoked faster. Or maybe it was deliberate that it does not get focused.

@ryyppy do you still remember?